portfolio.css

```diff
@@ -0,0 +1,79 @@
+/* Boilerplate Start */
+* {
+	box-sizing: border-box;
+}
+
+html {
+	font-size: 62.5%;
+}
+
+body {
+	margin: 0;
+	padding: 0;
+	font-size: 1.5rem;
+}
+/* Boilerplate End */
+
+/* Remove underline in links */
+a {
+	text-decoration: none;
+}
+
+/* John Hasznosi */
+.headertext {
+	overflow: auto;
+	text-align: center;
+	
+}
+
+/* John Hasznosi */
+.headertext h1 {
+	font-size: 2rem; 
+	font-family: "Charm", Times;
+	margin: 0;
+}
+
+/* logo */
+.headerimage {
+	text-align: center;
+}
+
+/* logo size */
+.headerimage img {
+	width: 200px;
+}
+
+/* navbar padding */
+nav {
+	overflow: auto;
+	padding: 15px;
+}
+
+/* nav container*/
+nav .link {
+	width: 100%;
```

Comment:
this actually does nothing

Reply:
Hi Mike. When I delete 'width: 100%' my navbar links buch up and move to the left.